packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts

```diff
@@ -192,8 +192,8 @@ export class PluginViewRegistry implements FrontendApplicationContribution {
         const iconClass = 'plugin-view-container-icon-' + viewContainer.id;
         const iconUrl = PluginSharedStyle.toExternalIconUrl(viewContainer.iconUrl);
         toDispose.push(this.style.insertRule('.' + containerClass + '.' + iconClass, () => `
-                mask: url('${iconUrl}') no-repeat 50% 50%;
-                -webkit-mask: url('${iconUrl}') no-repeat 50% 50%;
+                mask: url('${iconUrl}');
+                -webkit-mask: url('${iconUrl}');
```

Comment:
This was not a change in the original PR.

Reply:
Right, but it's the same and uses the same style.

Reply:
I am glad you're taking care of the code that much, but I would like to ask you to revert it. This change was not included in the previous PR. Let's focus on the followings:
 - fix the regression, and
 - revert the unnecessary `50% 50% / var(--theia-private-sidebar-icon-size) no-repeat` changes from the previous PR.

Once you have reverted this, we are ready to go. Your changes work fine. Please make sure you have squashed your commits; I would like to see one single commit for the PR.

Thank you!

Reply:
done

Reply:
OK, keeping two commits actually makes sense. You're a true hero to come back and fix it so quickly.🎖️ Thank you!